plugins/hypervisors/xenserver/src/com/cloud/hypervisor/XenServerGuru.java

```diff
@@ -58,7 +59,9 @@
 import com.cloud.vm.dao.UserVmDao;
 
 public class XenServerGuru extends HypervisorGuruBase implements HypervisorGuru, Configurable {
-    private final Logger LOGGER = Logger.getLogger(XenServerGuru.class);
+
+    private Logger logger = Logger.getLogger(getClass());
```

Comment:
why not static final LOG?

Reply:
The result is the same.

The variable was not declared as "static", but its name was LOGGER as a static variable. I only changed because at first I removed it as it was not being used. Then, when I added some logs, I re-added the variable and I created this way. I can change to static final.

I can change to static if you think it is better for some reason.